src/hooks/useMember/useMember.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import {
+  getMember as getMemberFn,
+  setMember as setMemberFn,
+  setActiveChannel as setActiveChannelFn,
+  removeMember as removeMemberFn,
+} from './utils'
+
+export type ActiveMember = {
+  id: string
+  activeChannelId: string
```

Comment:
I actually think we should make those nullable. For example when you log out as a member we will need to clear those. Our hook will allow this as well, either by exposing additional functions to unset or just allow empty values in setters